### ISSUE TITLE
bad gateway after session launch

### ISSUE THREAD

**Issue by drtobybrown** [opencadc/science-platform#408]
Session unavailable with `bad gateway` message for a few seconds after the link becomes active.

**Comment by brianmajor** [comment]
Yes, thanks for raising this.  There is a gap where this occurs while the Desktop container is initializing.

**Comment by drtobybrown** [comment]
This occurred with a notebook session

**Comment by at88mph** [comment]
Tracked in issue CADC-13931.

https://cadc-ccda.atlassian.net/browse/CADC-13931
